generate_response.py

```diff
@@ -603,7 +603,24 @@ def get_completion_starcoder_fim(
     print(completion)
     return completion
 
-
+def load_prompt_from_config(phase): # Added By Erfan
```

Comment:
I checked the access to args; because args is defined inside ##if __name__ == "__main__":##, The scope will be module level and all the functions inside that python file (module) will have access to it.